DTS-STN/Service-Canada-Labs
# Align fonts to Service Canada Design System guidelines

# Align fonts to Service Canada Design System guidelines

During a previous task I noticed that the font-size for the SCDS `<Heading>` component we use would change from 34px to 38px at viewport width of 768px. I referenced their [style guide](https://www.figma.com/file/YFz6bzcX1DSAQfu4BSfm6G/Extended-GC-Design-Library---SEDD?type=design&node-id=2-206&mode=design&t=zlQXNzRcmuEfkBNO-4) and setup the rest of our fonts to behave in the way laid out in the document.

## Test Instructions

1. Go to home page
2. View page under and above 768px display width and see that heading fonts behave responsively and that sizes match what is outlined [here](https://www.figma.com/file/YFz6bzcX1DSAQfu4BSfm6G/Extended-GC-Design-Library---SEDD?type=design&node-id=2-206&mode=design&t=zlQXNzRcmuEfkBNO-4)

## Definition of Done
